pkg/model/initializers.go

```diff
@@ -184,8 +187,13 @@ func (ml *ModelLoader) BackendLoader(opts ...Option) (client grpc.Backend, err e
 	if o.singleActiveBackend {
 		ml.mu.Lock()
 		log.Debug().Msgf("Stopping all backends except '%s'", o.model)
-		ml.StopAllExcept(o.model)
+		err := ml.StopAllExcept(o.model)
 		ml.mu.Unlock()
+		if err != nil {
+			log.Error().Err(err).Str("keptModel", o.model).Msg("error while shutting down all backends except for the keptModel")
```

Comment:
I'd lower this to a warning message and avoid to return in case of failure to avoid the LocalAI to be stuck here